Library/Formula/yaf.rb

```diff
@@ -1,18 +1,20 @@
-require 'formula'
-
 class Yaf < Formula
-  homepage 'http://tools.netsa.cert.org/yaf/'
-  url 'http://tools.netsa.cert.org/releases/yaf-2.4.0.tar.gz'
-  sha1 '0c5efb5543e61d0acd91b7e2b028d8f6d3497ae8'
+  homepage "https://tools.netsa.cert.org/yaf/"
+  url "https://tools.netsa.cert.org/releases/yaf-2.6.0.tar.gz"
+  sha1 "ac1ac70a8d1ed8b8dbbfd5ec803880a74fa2894b"
 
-  depends_on 'pkg-config' => :build
-  depends_on 'glib'
-  depends_on 'libfixbuf'
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+  depends_on "libfixbuf"
 
   def install
-    system './configure', "--disable-dependency-tracking",
+    system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/yaf", "--version"
```

Comment:
Looks like we could use http://tools.netsa.cert.org/yaf/libyaf/yaf_pcap.html and https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/test/fixtures/test.pcap to do something a bit more vigorous?